readme.md

```diff
@@ -65,7 +71,7 @@ All keys are strings. Values can be primitive types or composite types.
 ```
 
 #### Array
-Array is a list of values separated by a newline `\n`. Its enclosed in square brackets `[]`.
+Array is a list of values separated by a newline (`\n`), enclosed in square brackets (`[]`).
```

Comment:
Should this be `primitive` values, or would it be possible to have a list of multimaps?

Reply:
Yes, This should possible. Here is an example - 

```groovy
person: {
  name: 'Bruno'
  links: [
    {
      site: 'github'
      url: 'https://www.github.com/usebruno/bruno'
    }
    {
      site: 'twitter'
      url: 'https://www.twitter.com/use_bruno'
      description: (
        We are on twitter.
        Follow us to get updates about Bruno.
      )
    }
  ]
}
```

Reply:
Which also means that nested arrays are allowed. Worth calling out in the syntax description.